### PR TITLE
Bug fixes for PermissionsViewer

### PR DESCRIPTION
**Pull request by Allvaa**
fix issue #278. proof: 
image
 image
 and fixed this 
image

### CODE REVIEW

**Comment by rauenzi** [comment]
Escaping html is important because someone could make their nickname `<script>alert("something")` and it would execute javascript on the client. I cannot accept this change as-is.

**Comment by Allvaa** [comment]
well then, i found another way to fix it without removing escapeHTML

File: Plugins/PermissionsViewer/PermissionsViewer.plugin.js [hunk 1]
```diff
@@ -635,8 +636,9 @@ module.exports = (() => {
                 const item = DOMTools.createElement(!isOverride || displayRoles[role].type == 0 ? this.modalButton : Utilities.formatTString(this.modalButtonUser, {avatarUrl: user.avatarURL}));
                 if (!isOverride || displayRoles[role].type == 0) item.style.color = referenceRoles[role].colorString;
                 else item.style.color = member.colorString;
-                if (isOverride) item.querySelector(".role-name").textContent = escapeHTML(displayRoles[role].type == 0 ? referenceRoles[role].name : user.username);
-                else item.querySelector(".role-name").textContent = escapeHTML(referenceRoles[role].name);
+                console.log(referenceRoles[role].name)
```
Comment (rauenzi) [inline Plugins/PermissionsViewer/PermissionsViewer.plugin.js:639]: Please remove this

**Review by rauenzi — COMMENTED** [review]
Can you also make the corresponding changes in the source files?

File: Plugins/PermissionsViewer/PermissionsViewer.plugin.js (hunk 1 above)
Comment (Allvaa) [reply Plugins/PermissionsViewer/PermissionsViewer.plugin.js]: forgot to remove it sorry. also already did it

**Comment by rauenzi** [comment]
By the source files I meant, the source from which `PermissionsViewer.plugin.js` is transpiled... See here https://github.com/rauenzi/BetterDiscordAddons/tree/master/src/plugins/PermissionsViewer

**Comment by Allvaa** [comment]
like this?

**Comment by Allvaa** [comment]
hello?

**Pull request merged**